tsconfig.json

```diff
@@ -7,11 +7,15 @@
     "module": "ES2022",
     "target": "ES2022",
     "lib": ["ES2022", "dom"],
+    "allowSyntheticDefaultImports": true,
     "declaration": false,
     "downlevelIteration": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "skipLibCheck": true,
-    "sourceMap": true
+    "sourceMap": true,
+    "paths": {
+      "@igo2/*": ["node_modules/@igo2/*", "../../packages/*/src/public_api"]
```

Comment:
Workaround, when we refer to the symbolic link, we need to provide the source code for Typescript module resolution. We fallback on the last url if it exists.